package.json

```diff
@@ -23,7 +23,7 @@
     "@boxyhq/metrics": "0.2.5",
     "@boxyhq/react-ui": "3.3.13",
     "@boxyhq/saml-jackson": "1.13.2",
-    "@heroicons/react": "2.0.18",
+    "@heroicons/react": "^2.0.18",
     "@hookform/resolvers": "3.3.1",
```

Comment:
Heroicon is already installed in the project, can you revert the changes to package.json/package-lock.json?